admin_guide/iptables.adoc

```diff
@@ -32,6 +32,13 @@ not impact the operation of {product-title} and the Docker service. Also, change
 will often need to be made on all nodes in the cluster. Use caution, as iptables
 is not designed to have multiple concurrent users and it is very easy to break
 {product-title} and Docker networking.
+
+{product-title} provides one chain which is specifically intended for
```

Comment:
@danwinship @ahardin-rh I'd agree with Ashley. Commas are really only for separating clauses. I'd suggest "OpenShift provides a number of chains, one of which is specifically intended for..."

I was asked to have a look at this PR and see if it needed a followup and this is the only thought I had. I'll do that.